go.mod

```diff
@@ -1,17 +1,25 @@
 module github.com/AllenDang/giu
 
-go 1.16
+go 1.17
```

Comment:
Why is it necessary? IMO it is better to keep support for older go versions as long as possible

Reply:
oh, I'm sorry I've just read pr description :grinning: